ormconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "type": "postgres",
+    "host": "127.0.0.1",
+    "port": 5432,
+    "username": "postgres",
+    "password": "postgres",
+    "database": "test",
+    "entities": [
+        "dist/**/*.entity.js"
+    ],
+    "synchronize": true
+}
```

Comment:
Reminder: find a way to avoid duplication (with the env var defined in docker). For example by creating a common file used with or without docker.